pkg/node/node.go

```diff
@@ -239,6 +227,38 @@ func NewBee(o Options) (*Bee, error) {
 		b.debugAPIServer = debugAPIServer
 	}
 
+	var wg sync.WaitGroup
+
+	// TODO: be more resilient on connection errors and connect in parallel
+	for _, a := range o.Bootnodes {
+		wg.Add(1)
+		go func(aa string) {
+			defer wg.Done()
+			addr, err := ma.NewMultiaddr(aa)
+			if err != nil {
+				logger.Debugf("multiaddress fail %s: %w", aa, err)
+				logger.Errorf("connect to bootnode %s: %w", aa, err)
+				return
+			}
+
+			overlay, err := p2ps.Connect(p2pCtx, addr)
+			if err != nil {
+				logger.Debugf("connect fail %s: %w", aa, err)
+				logger.Errorf("connect to bootnode %s: %w", aa, err)
```

Comment:
if you're logging the exact same things, why do we need to log it both in debug and in error?